# This file is part of PerlIO-via-EscStatus.
#
# PerlIO-via-EscStatus is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License as published
# by the Free Software Foundation; either version 3, or (at your option) any
# later version.
#
# PerlIO-via-EscStatus is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
# Public License for more details.
#
# You should have received a copy of the GNU General Public License along
# with PerlIO-via-EscStatus.  If not, see <http://www.gnu.org/licenses/>.

# include /usr/share/cdbs/1/rules/simple-patchsys.mk


diff -Nur -x '*.orig' -x '*~' escstatus/lib/ProgressMonitor/Stringify/ToEscStatus.pm escstatus.new/lib/ProgressMonitor/Stringify/ToEscStatus.pm
--- escstatus/lib/ProgressMonitor/Stringify/ToEscStatus.pm	2010-01-23 16:35:02.000000000 +1100
+++ escstatus.new/lib/ProgressMonitor/Stringify/ToEscStatus.pm	2010-01-23 16:39:04.000000000 +1100
@@ -140,8 +140,8 @@
 
 giving the output form EscStatus uses.  The contents of the status line are
 built by the configured C<ProgressMonitor> field objects in the usual way.
-See F<examples/progressmonitor.pl> in the EscStatus sources for a complete
-program.
+See F</usr/share/doc/libperlio-via-escstatus-perl/examples/progressmonitor.pl>
+for a complete program.
 
 =head1 FUNCTIONS
 
